Src/Newtonsoft.Json-for-Unity.Tests/Assets/SystemWebPolyfill/HttpUtility.cs

```diff
@@ -0,0 +1,670 @@
+        public static void HtmlEncode(string s, TextWriter output)
+        {
+            if (output == null) {
+                throw new ArgumentNullException ("output");
+            }
+
+            if (!String.IsNullOrEmpty (s)) {
+                HttpEncoder.Current.HtmlEncode (s, output);
+            }
+        }
+        public static string HtmlEncode (object value)
+        {
+            if (value == null)
+                return null;
+
+#if !(MOBILE || NO_SYSTEM_WEB_DEPENDENCY || true)
+            IHtmlString htmlString = value as IHtmlString;
+            if (htmlString != null)
+                return htmlString.ToHtmlString ();
+#endif
+
+            return HtmlEncode (value.ToString ());
+        }
+
+        public static string JavaScriptStringEncode (string value)
+        {
+            return JavaScriptStringEncode (value, false);
+        }
+
+        public static string JavaScriptStringEncode (string value, bool addDoubleQuotes)
+        {
+            if (String.IsNullOrEmpty (value))
+                return addDoubleQuotes ? "\"\"" : String.Empty;
+
+            int len = value.Length;
+            bool needEncode = false;
+            char c;
+            for (int i = 0; i < len; i++) {
+                c = value [i];
+
+                if (c >= 0 && c <= 31 || c == 34 || c == 39 || c == 60 || c == 62 || c == 92) {
+                    needEncode = true;
+                    break;
+                }
+            }
+
+            if (!needEncode)
+                return addDoubleQuotes ? "\"" + value + "\"" : value;
+
+            var sb = new StringBuilder ();
+            if (addDoubleQuotes)
+                sb.Append ('"');
+
+            for (int i = 0; i < len; i++) {
+                c = value [i];
+                if (c >= 0 && c <= 7 || c == 11 || c >= 14 && c <= 31 || c == 39 || c == 60 || c == 62)
+                    sb.AppendFormat ("\\u{0:x4}", (int)c);
+                else switch ((int)c) {
+                        case 8:
+                            sb.Append ("\\b");
+                            break;
+
+                        case 9:
+                            sb.Append ("\\t");
+                            break;
+
+                        case 10:
+                            sb.Append ("\\n");
+                            break;
+
+                        case 12:
+                            sb.Append ("\\f");
+                            break;
+
+                        case 13:
+                            sb.Append ("\\r");
+                            break;
+
+                        case 34:
+                            sb.Append ("\\\"");
+                            break;
+
+                        case 92:
+                            sb.Append ("\\\\");
+                            break;
+
+                        default:
+                            sb.Append (c);
+                            break;
+                    }
+            }
+
+            if (addDoubleQuotes)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity/pullRequest?prid=4755183)